src/View/nav.html.twig

```diff
@@ -0,0 +1,52 @@
+<nav role='navigation'>
+	<div class="main">
```

Comment:
Could you transform this div tag into an Ul since it will be a list of links, and define each child element as a li